components/chainhook-sdk/src/indexer/stacks/mod.rs

```diff
@@ -253,8 +253,12 @@ pub fn standardize_stacks_block(
     let pox_cycle_length: u64 = (chain_ctx.pox_info.prepare_phase_block_length
         + chain_ctx.pox_info.reward_phase_block_length)
         .into();
-    let current_len =
-        block.burn_block_height - (1 + chain_ctx.pox_info.first_burnchain_block_height);
+    let current_len = if chain_ctx.pox_info.first_burnchain_block_height >= block.burn_block_height
+    {
+        0
+    } else {
+        block.burn_block_height - (1 + chain_ctx.pox_info.first_burnchain_block_height)
```

Comment:
Thanks for catching this @MicaiahReid! could we use `saturating_sub` instead?

Reply:
Nice, I didn't know about that! Done!